13-managing-state/projects/final/pubspec.yaml

```diff
@@ -12,7 +12,6 @@ dependencies:
   flutter:
     sdk: flutter
 
-  path: ^1.8.2
```

Comment:
Why?

Reply:
It's not described in the chapter. Plus it’s a transient dependency of some other package, so it’s included anyway. If it's needed in subsequent chapters we’ll explicitly tell the reader to add it.